consensus/blake3pow/consensus.go

```diff
@@ -254,6 +254,15 @@ func (blake3pow *Blake3pow) verifyHeader(chain consensus.ChainHeaderReader, head
 	if expected.Cmp(header.Difficulty()) != 0 {
 		return fmt.Errorf("invalid difficulty: have %v, want %v", header.Difficulty(), expected)
 	}
+
+	// If the chain is not zone, check the EntropyThreshold calculation
+	if nodeCtx != common.ZONE_CTX {
```

Comment:
conversely, the difficulty check should only occur if `nodeCtx == common.ZONE_CTX`. Presently that isn't the case.

Reply:
Quai Network is a new design for building more efficient blockchains. The project is based on developments in blockchain sharding technologies and the innovative system of Proof-of-Work 2.0. Quai Network can scale horizontally to accommodate more users as the network grows. Quai Network aims to increase participation in Web3 technology by taking innovative steps in overall blockchain scalability.